src/otx/core/data/transform_libs/utils.py

```diff
@@ -104,14 +104,29 @@ def __get__(self, obj, cls):  # noqa: ANN001
         return copy.copy(self)
 
 
-def to_np_image(img: Tensor) -> np.ndarray:
+def get_image_shape(img: np.ndarray | Tensor | list) -> tuple[int, int]:
+    """Get image(s) shape with (height, width)."""
+    if isinstance(img, np.ndarray):
+        return img.shape[:2]
```

Comment:
Are you encountering the mypy error `Incompatible return value type (got "tuple[int, ...]", expected "tuple[int, int]")`? I suggest explicitly returning it as `return (img.shape[0], img.shape[1])`

Reply:
I updated this PR with the latest develop and fixed mypy error about `to_np_image`, not `get_image_shape`, at https://github.com/openvinotoolkit/training_extensions/pull/3644/commits/0db74310b56efc377e0bace0a766226a2e5aa331.